usaspending_api/search/delta_models/award_search.py

```diff
@@ -1,126 +1,131 @@
+    "generated_unique_award_id": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "display_award_id": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "update_date": {"delta": "TIMESTAMP", "postgres": "TIMESTAMP", "gold": False},
+    "certified_date": {"delta": "DATE", "postgres": "DATE", "gold": True},
+    "create_date": {"delta": "TIMESTAMP", "postgres": "TIMESTAMP", "gold": True},
+    "piid": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "fain": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "uri": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "parent_award_piid": {"delta": "STRING", "postgres": "TEXT", "gold": True},
+    "award_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": False},
+    "total_obligation": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": False},
+    "description": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "total_obl_bin": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "total_subsidy_cost": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": False},
+    "total_loan_value": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": False},
+    "total_funding_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23,2)", "gold": True},
+    "total_indirect_federal_sharing": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": True},
+    "base_and_all_options_value": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": True},
+    "base_exercised_options_val": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": True},
+    "non_federal_funding_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": True},
+    "recipient_hash": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "recipient_levels": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]", "gold": False},
+    "recipient_name": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "raw_recipient_name": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "recipient_unique_id": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "parent_recipient_unique_id": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "recipient_uei": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "parent_uei": {"delta": "STRING", "postgres": "TEXT", "gold": False},
+    "business_categories": {"delta": "ARRAY<STRING>", "postgres": "TEXT[]", "gold": False},
+    "total_subaward_amount": {"delta": "NUMERIC(23, 2)", "postgres": "NUMERIC(23, 2)", "gold": True},
+    "subaward_count": {"delta": "INTEGER", "postgres": "INTEGER", "gold": True},
+    "action_date": {"delta": "DATE", "postgres": "DATE", "gold": False},
+    "fiscal_year": {"delta": "INTEGER", "postgres": "INTEGER", "gold": False},
+    "last_modified_date": {"delta": "DATE", "postgres": "DATE", "gold": True},
```

Comment:
I believe this field was on the original model

Reply:
fixed